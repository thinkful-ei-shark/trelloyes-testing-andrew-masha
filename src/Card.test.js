import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Card from './Card.js';

it('renders without crashing', () => {
  const div = document.createElement('div');

  ReactDOM.render(<Card />, div);
  ReactDOM.unmountComponentAtNode(div);
})

it('renders as expected', () => {
  const tree = renderer
    .create(<Card 
      title="this is title" 
      content="this is content" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});