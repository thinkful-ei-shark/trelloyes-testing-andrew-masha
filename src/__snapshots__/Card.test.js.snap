// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    this is title
  </h3>
  <p>
    this is content
  </p>
</div>
`;
